--- Frodo/Src/Display.h.orig	Wed Aug  6 20:56:26 1997
+++ Frodo/Src/Display.h	Sat Oct 12 19:39:02 2002
@@ -112,7 +112,7 @@
 
 #ifdef __unix
 	void draw_led(int num, int state);	// Draw one LED
-	static void pulse_handler(...);		// LED error blinking
+	static void pulse_handler(int dummy);	// LED error blinking
 #endif
 
 #ifdef WIN32
--- Frodo/Src/Display_x.i.orig	Sat Oct 12 19:33:54 2002
+++ Frodo/Src/Display_x.i	Sat Oct 12 19:37:13 2002
@@ -555,7 +555,7 @@
  *  LED error blink
  */
 
-void C64Display::pulse_handler(...)
+void C64Display::pulse_handler(int dummy)
 {
 	for (int i=0; i<4; i++)
 		switch (c64_disp->led_state[i]) {
--- Frodo/Src/SAM.cpp.orig	Wed Aug  6 20:56:30 1997
+++ Frodo/Src/SAM.cpp	Sat Oct 12 19:37:24 2002
@@ -470,7 +470,7 @@
 struct sigaction my_sa;
 #endif
 
-static void handle_abort(...)
+static void handle_abort(int dummy)
 {
 	WasAborted = true;
 #if !defined(HAVE_SIGACTION) && defined(HAVE_SIGNAL)
--- Frodo/Src/CmdPipe.h.orig	Wed Aug  6 20:56:26 1997
+++ Frodo/Src/CmdPipe.h	Sat Oct 12 19:40:17 2002
@@ -82,4 +82,4 @@
 
 };
 
-#endif CmdPipe_h
+#endif /* CmdPipe_h */
--- Frodo/Src/CmdPipe.cpp.orig	Wed Aug  6 20:56:26 1997
+++ Frodo/Src/CmdPipe.cpp	Sat Oct 12 19:41:21 2002
@@ -13,6 +13,7 @@
 	#include <unistd.h>
 	#include <sys/wait.h>
 	#include <sys/types.h>
+	#include <stdlib.h>
 	#include <string.h>
 	#include <signal.h>
 
