diff -urN Frodo-4.1b.org/Src/C64_x.i Frodo-4.1b/Src/C64_x.i
--- Frodo-4.1b.org/Src/C64_x.i	Sat Feb 22 15:26:13 2003
+++ Frodo-4.1b/Src/C64_x.i	Sat Feb 22 15:28:37 2003
@@ -100,7 +100,7 @@
 	gui = 0;
 #else
 	// try to start up Tk gui.
-	gui = new CmdPipe("wish", "TkGui.tcl");
+	gui = new CmdPipe("/usr/bin/wish", FRODO_HOME "TkGui.tcl");
 	if (gui) {
 		if (gui->fail) {
 			delete gui; gui = 0;
diff -urN Frodo-4.1b.org/Src/Makefile.in Frodo-4.1b/Src/Makefile.in
--- Frodo-4.1b.org/Src/Makefile.in	Sat Feb 22 15:26:14 2003
+++ Frodo-4.1b/Src/Makefile.in	Sat Feb 22 15:29:04 2003
@@ -8,7 +8,7 @@
 @SET_MAKE@
 CXX    = @CXX@
 LIBS   = @LIBS@
-CFLAGS = @CFLAGS@ -I./ -DFRODO_HPUX_REV=@HPUX_REV@ -DKBD_LANG=@KBD_LANG@
+CFLAGS = @CFLAGS@ -I./ -DFRODO_HPUX_REV=@HPUX_REV@ -DKBD_LANG=@KBD_LANG@ -DFRODO_HOME=$(FRODOHOME)
 
 INSTALL         = @INSTALL@
 INSTALL_PROGRAM = @INSTALL_PROGRAM@
diff -urN Frodo-4.1b.org/Src/Prefs.cpp Frodo-4.1b/Src/Prefs.cpp
--- Frodo-4.1b.org/Src/Prefs.cpp	Sat Feb 22 15:26:14 2003
+++ Frodo-4.1b/Src/Prefs.cpp	Sat Feb 22 15:28:37 2003
@@ -39,7 +39,7 @@
 	for (int i=0; i<4; i++)
 		DriveType[i] = DRVTYPE_DIR;
 
-	strcpy(DrivePath[0], "64prgs");
+	strcpy(DrivePath[0], FRODO_HOME "64prgs");
 	strcpy(DrivePath[1], "");
 	strcpy(DrivePath[2], "");
 	strcpy(DrivePath[3], "");
diff -urN Frodo-4.1b.org/Src/main.cpp Frodo-4.1b/Src/main.cpp
--- Frodo-4.1b.org/Src/main.cpp	Sat Feb 22 15:26:14 2003
+++ Frodo-4.1b/Src/main.cpp	Sat Feb 22 15:28:37 2003
@@ -24,10 +24,10 @@
 #define CHAR_ROM_FILE	"FrodoRsrc:Char_ROM"
 #define FLOPPY_ROM_FILE	"FrodoRsrc:1541_ROM"
 #else
-#define BASIC_ROM_FILE	"Basic ROM"
-#define KERNAL_ROM_FILE	"Kernal ROM"
-#define CHAR_ROM_FILE	"Char ROM"
-#define FLOPPY_ROM_FILE	"1541 ROM"
+#define BASIC_ROM_FILE	FRODO_HOME "Basic ROM"
+#define KERNAL_ROM_FILE	FRODO_HOME "Kernal ROM"
+#define CHAR_ROM_FILE	FRODO_HOME "Char ROM"
+#define FLOPPY_ROM_FILE	FRODO_HOME "1541 ROM"
 #endif
 
 
