--- Frodo/Src/configure.in.orig	Tue Feb 27 01:00:16 2001
+++ Frodo/Src/configure.in	Tue Feb 27 01:05:46 2001
@@ -34,14 +34,8 @@
 
 if [[ "$ac_cv_header_linux_joystick_h" = "yes" ]]; then
   AC_MSG_CHECKING(whether linux/joystick.h is broken)
-  if grep "#include" /usr/include/linux/joystick.h >/dev/null; then
-    AC_MSG_RESULT(yes)
-    BROKEN_JOYSTICK_H=1
-    grep -v "#include" /usr/include/linux/joystick.h >joystick.h
-  else
-    AC_MSG_RESULT(no)
-    BROKEN_JOYSTICK_H=0
-  fi
+  AC_MSG_RESULT(no)
+  BROKEN_JOYSTICK_H=0
 fi
 
 AC_CHECK_SIZEOF(char)
